verification/tests/settings.rs

```diff
@@ -0,0 +1,11 @@
+use pretty_assertions::assert_eq;
+use verification::Settings;
+
+#[test]
+#[cfg(target_os = "linux")]
+fn test_example_settings() {
```

Comment:
maybe add `#[ignore]` to this test and test it with `cargo test -- --include-ignored` ?

Reply:
but why it depends on os?...

Reply:
@leviska Because default `solidity.fetcher.list.list_url` is different for linux and macos/windows systems

Reply:
let's make this test ignore this field, or set it manually to the OS default, or check this field only on linux, and ignore only the field on other systems, not the whole test. 
I think there is no real reason why we can't test the config on the macos or windows, and one different line is not an excuse

Reply:
Made them run on other systems as well without checking `list_url`